tests/integration/functional/steps/test_log_artifact_metadata.py

```diff
@@ -0,0 +1,92 @@
+"""Integration tests for `zenml.log_artifact_metadata()`."""
```

Comment:
I would love to see a test for an unnamed multiple-artifact case.

Reply:
ah I actually implemented a test case for this already at the bottom, just forgot to name it `test_`

Reply:
I'm talking about these standard names like `output_1`, etc.